### PR TITLE
Backend Roadmap - a collection of improvements

### PR DESCRIPTION
**Pull request by leni-kirilov**

### CODE REVIEW

File: src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ Even if you’re a beginner the least you would have known is that Web Developme
 In the same way, when it comes to Backend Web Development – we primarily require a backend (or you can say server-side) programming language to make the website function along with various other tools & technologies such as databases, frameworks, web servers, etc.
 
 Pick a language from the given list and make sure to learn its quirks, core details about its runtime e.g. concurrency, memory model etc.
+
+[@article@ Top Languages for job ads](https://www.tiobe.com/tiobe-index/)
```
Comment (leni-kirilov) [inline src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md:9]: I disagree that this list doesn't include the most famous or sought-after languages in the backend (web) world - like Java or C/C++.

Javascript is way less popular than Python, Java in the backend world

Java also has Frontend development capabilities, but that doesn't mean it's most popular and it should be put in the Frontend roadmap

File: src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md [hunk 2]
```diff
@@ -5,3 +5,5 @@ Even if you’re a beginner the least you would have known is that Web Developme
 In the same way, when it comes to Backend Web Development – we primarily require a backend (or you can say server-side) programming language to make the website function along with various other tools & technologies such as databases, frameworks, web servers, etc.
```
Comment (leni-kirilov) [inline src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md:5]: the names of the files in this folder are strange 🤔

File: src/data/roadmaps/backend/content/repo-hosting-services@NvUcSDWBhzJZ31nzT4UlE.md [hunk 3]
```diff
@@ -7,4 +7,4 @@ Visit the following resources to learn more:
 - [@opensource@GitHub](https://github.com/features/)
 - [@article@GitLab](https://about.gitlab.com/)
 - [@article@BitBucket](https://bitbucket.org/product/guides/getting-started/overview)
-- [@article@How to choose the best source code repository](https://bitbucket.org/product/code-repository)
+- [@article@How to choose the best source code repository](https://blockandcapital.com/en/choose-code-repository/)
```
Comment (leni-kirilov) [inline src/data/roadmaps/backend/content/repo-hosting-services@NvUcSDWBhzJZ31nzT4UlE.md:10]: unbiased blogpost (that is also outdated as Github has integrated CI in githubActions)

**Comment by leni-kirilov** [comment]
I really liked the "guides" about Authentication. Nice diagrams
I feel that a few important topics are ignored : Performance, Monitoring, REPL , Dev Tools, 
Should I create follow-up to create nodes?

**Comment by leni-kirilov** [comment]
When I finish the expert/detailed roadmap, I'll remove the draft mode again

File: src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md (hunk 2 above)
Comment (dansholds) [reply src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md:5]: @leni-kirilov this is because it's been converted to the new editor.

Comment (leni-kirilov) [reply src/data/roadmaps/backend/content/pick-a-language@2f0ZO6GJElfZ2Eis28Hzg.md:5]: where can I read more about this new editor? and how to use it or even contribute?

**Comment by leni-kirilov** [comment]
@dansholds I'd say I'm not going to contribute to this one much more soon. I'm looking into other things

**Comment by dansholds** [comment]
Ok awesome, I'll give it a look and get it merged

**Comment by ayomidethomax** [comment]
How can I get connected to you guys

**Review by kamranahmedse — APPROVED** [review]

**Comment by kamranahmedse** [comment]
Thanks @leni-kirilov — really helpful!

**Pull request merged**